ddht/v5_1/network.py

```diff
@@ -404,10 +489,10 @@ async def _fetch_enr(
     async def recursive_find_nodes(self, target: NodeID) -> Tuple[ENRAPI, ...]:
         return await common_recursive_find_nodes(self, target)
 
-    async def get_nodes_near(
+    async def explore_target(
```

Comment:
nit: i understand the reasoning in the PR description, but `explore_target` is pretty mysterious as to what return values to expect. haven't had any ideas i love - most are similar to the name it replaced. maybe just `get/find_nodes` if you think the `near` part is misleading. `near` still feels correct in the relative sense; i don't think there's necessarily an implied promise of `nearest`.